_rules/html-lang-valid-bf051a.md

```diff
@@ -19,11 +19,13 @@ authors:
 
 ## Applicability
 
-The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute that is not empty ("").
+The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element with a `lang` and/or `xml:lang` attribute that is not empty ("").
+
+**Note**: Documents embedded into other documents, such as through `iframe` or `object` elements are not applicable because they are not web pages according to the definition in WCAG.
 
 ## Expectation
 
-The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is not empty ("").
+For each test target, the `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is not empty ("").
```

Comment:
This expectation should get the same wording as the one you did in https://github.com/act-rules/act-rules.github.io/blob/0d3d29695abbffa7c09eda289ae72f72aaf2ad8a/_rules/lang-valid-de46e4.md (**unless** the attribute is empty).
And does have the same issue about the non-existing attribute.